src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -36,8 +36,10 @@
 
 {% block form_label -%}
   {% spaceless %}
-    {% if label is same as(false) %}
+    {% if label is same as(null) %}
       <div class="{{ block('form_label_class') }}"></div>
+    {% elseif label is same as(false) %}
+      {# Use false as label is you don't want an empty column #}
```

Comment:
this condition could be removed (you'd test for different than false instead of the `else` part... except if you kept this empty conditional block just for the comment ?

Reply:
The condition and the empty space is wanted, when `false` is explicitely used I want:
- no label
- no empty side column

Only the widget itself is kept in "full width"

By the way the default value (when nothing is set) is `null` which matches the previous behaviour to always have an empty column, so the new `false` option is when you want a large widget